docs/posta-maskeleme.md

```diff
@@ -0,0 +1,44 @@
+# E-Posta Maskeleme
+
+**E-posta maskeleme**, gerçek e-posta adresinizi gizleyerek koruma altına alma yöntemidir. Bu teknik, e-posta adreslerinizin spam, kimlik hırsızlığı ve diğer kötü niyetli faaliyetlerden korunmasına yardımcı olur. E-posta makselemenin bazı yolları şunlardır.
```

Comment:
Şimdi aklıma şey geldi. Eposta maskelememiz neden önemli? Çünkü ifşa olan adres ile sizin bağlantınızı kurarlarsa diğer hesaplarınızıda aynı şifre ve benzeri metrikleri kullanabilirler. Bu şekilde hesaplarınız, epostalarınız ve sizin aranızda bir bağlantı kurulamıyor ve kullanımınıza bağlı olarak her üyelik/eposta kendine özgü oluyor. asd124zxy@addy adlı bir eposta ile kullanıcıadı@gmail adlı eposta arasındaki bağlantı hiçbir zaman için (siz belirtmedikçe) kurulamıyor ve kullanıcının güvenliği bundan olumlu etkileniyor.